study.md

```diff
@@ -93,15 +118,17 @@ Ruby:
 ```ruby
 array = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
 
-# your answer here
+array.map { |array| array ** 2  }
 ```
```

Comment:
you don't have to call it array_element. You can call it anything you want, but just know what it REPRESENTS